core_sign_update

```diff
@@ -63,6 +66,14 @@ for key in "${private_keys[@]}"; do
 done
 signature_sizes="${signature_sizes:1:${#signature_sizes}}"
 
+IFS=":" read -a user_signatures <<< "$FLAGS_user_signatures"
```

Comment:
a comment explaining the bash manipulation going on would be useful

Reply:
I used `read` here to match the way `$FLAGS_private_keys`/`$FLAGS_public_keys` are handled elsewhere in this script, but I don't have a preference between the two.